app/assets/javascripts/income_calculator.coffee

```diff
@@ -1,26 +1,42 @@
+# set values
 $min_val = 1085
 $pp_child = 245
 $couple_supp = 160
 
+# global variables
+$low_threshold = 0
+$high_threshold = 0
+$income = 0
+
 calculate = ->
   $children_val = $('#children').val() * $pp_child
   $add_single_supp = if $('input:radio[name=couple]').val() == 'false' then 0 else $couple_supp
-  $check_val = $('#income').val() - $min_val + $children_val + $add_single_supp
   $curr_fee = parseFloat($('#fee').val())
-  $max_to_pay = Math.min(Math.max(Math.floor($check_val / 10) * 10 * 0.5, 0), $curr_fee)
-  $remittance = Math.max($curr_fee - $max_to_pay, 0)
-  if $check_val > 4000
+  $income = $('#income').val()
+  $low_threshold = $min_val + $children_val + $add_single_supp
+  $high_threshold = $low_threshold + Math.min(4000, $curr_fee * 2)
+  if fullRemittance()
```

Comment:
If you don't plan to reuse this it's prob easier to read as:
`if $income < $low_threshold`